packages/react-examples/src/react-accordion/Accordion/Accordion.stories.tsx

```diff
@@ -2,35 +2,71 @@ import * as React from 'react';
 import { AccordionItem, AccordionHeader, AccordionPanel, Accordion, AccordionProps } from '@fluentui/react-accordion';
 import { ArgType } from '@storybook/addons';
 import { RocketIcon } from '@fluentui/react-icons-mdl2';
+import { Button } from '@fluentui/react-button';
+
+const placeholder: React.CSSProperties = {
+  borderRadius: '2px',
+  background: '#e0e0e0',
+  marginBottom: '12px',
+  height: '13px',
+};
+
+const placeholderSmall: React.CSSProperties = {
+  ...placeholder,
+  width: '30%',
+};
+
+const placeholderSquare: React.CSSProperties = {
+  borderRadius: '8px',
+  height: '80px',
+  width: '80px',
+  marginRight: '8px',
+  background: '#e0e0e0',
+  display: 'inline-block',
+};
+
+const placeholderWithButton: React.CSSProperties = {
+  height: '42px',
+  background: '#EDEBE9',
+  padding: '12px',
+  borderRadius: '2px',
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'space-around',
+  flexWrap: 'wrap',
+  overflow: 'auto',
+  minWidth: '320px',
+};
 
 export const AccordionExample = (props: AccordionProps) => {
   return (
     <Accordion {...props} icon={props.icon ? <RocketIcon /> : undefined}>
       <AccordionItem>
         <AccordionHeader>Accordion Header 1</AccordionHeader>
         <AccordionPanel>
-          <div>
-            <button>Button 1</button>
```

Comment:
I guess the buttons were there to test keyboard navigation

Reply:
Interesting. I put in `Button` components in the 3rd Accordion Panel, and Tab doesn't navigate to them automatically. Do I need need to manually put a tab index on them?

Reply:
For what I'm testing Tab **is** navigating to the internal button element, there's just no style being added, that's defientely not a bug, simply not supported at the moment, this should be fixed by https://github.com/microsoft/fluentui/pull/18081

No need for tabindex @paulgildea, this will be fixed shortly!

Reply:
This PR should not worry about this!!!! this will be solved by a separate PR since it's a totally unrelated issue